test/data/conformance/no-financial.xml

```diff
@@ -96,6 +63,12 @@
           <eeSummary:radiationExposureInd>true</eeSummary:radiationExposureInd>
         </eeSummary:specialFactors>
       </eeSummary:enrollmentDeterminationInfo>
+      <eeSummary:financialsInfo xmlns:eeSummary="http://jaxws.webservices.esr.med.va.gov/schemas">
```

Comment:
Yep, that's exactly how I did it - these conformance tests are totally supposed to "catch" these sort of logic changes, and let you take a step back and look at how different applications change in response to things like the above change in enrollment-systems.